src/cfml/system/modules_app/server-commands/commands/server/prune.cfc

```diff
@@ -0,0 +1,159 @@
+/**
+ * Prune.
+ * .
+ * {code:bash}
+ * server prune months=6
+ * server prune months=6 --list
+ * server prune months=6 --force
+ * {code}
+ **/
+component {
+
+	// di
+	property name="serverService" inject="ServerService";
+
+	/**
+	 * Prune.
+	 *
+	 * @months.hint forget servers which last started date is greater or equal to months you set
+     * @list.hint give a list of servers which last started date is greater or equal to the months you set
+	 **/    
+    function run(
```

Comment:
What happened to the `--JSON` flag we discussed in Slack?  I don't see it implemented here.

Reply:
![image](https://user-images.githubusercontent.com/80276017/124818246-07f33f80-df28-11eb-92d9-dc85f2e060e9.png)